packages/cypher-builder/src/expressions/procedures/db.ts

```diff
@@ -58,14 +59,20 @@ export class FullTextQueryNodes extends Clause {
 
     public getCypher(env: CypherEnvironment): string {
         const targetId = env.getReferenceId(this.targetNode);
+        let scoreYield = "";
+        if (this.scoreVar) {
+            const scoreId = env.getReferenceId(this.scoreVar);
+            scoreYield = `, score AS ${scoreId}`;
+        }
```

Comment:
We have a neat helper for this:
```typescript
const scoreYield = compileCypherIfExists(this.scoreVar, env, { prefix: ", score AS "});
```

This will already handle the logic of pretending `score AS` only if `scoreVar` is defined.

Just for consistency sake, the line above (`env.getReferenceId`) should be:
```ts
const targetId = this.targetNode.getCypher(env);
```